rest-client/pom.xml

```diff
@@ -154,6 +154,30 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
This make IDE tooling nicer, with javadocs and actual source instead of decompiled classes